src/wallet/address.ts

```diff
@@ -480,7 +494,11 @@ const convertBtcPrivateKey = (testnet: boolean, privkey: string) => {
     const keyPair = ECPair.fromWIF(privkey, network);
     return payments.p2pkh({pubkey: keyPair.publicKey, network}).address as string;
 };
-
+const convertQTUMPrivateKey = (testnet: boolean, privkey: string) => {
+    const network = testnet ? QTUM_NETWORK_TESTNET : QTUM_NETWORK_MAINNET;
+    const keyPair = ECPair.fromWIF(privkey, network);
+    return payments.p2pkh({pubkey: keyPair.publicKey, network}).address as string;
```

Comment:
Is it possible to use segwit address here, not p2pkh? Does qtum supports that?

Reply:
as discussed using p2pkh